infra/Pos/Slotting/Impl/Simple.hs

```diff
@@ -42,49 +51,49 @@ data SimpleSlottingState = SimpleSlottingState
     { _sssLastSlot :: !SlotId
     }
 
-type SimpleSlottingVar = TVar SimpleSlottingState
+type SimpleSlottingStateVar = TVar SimpleSlottingState
 
-mkSimpleSlottingVar :: (MonadIO m, HasConfiguration) => m SimpleSlottingVar
-mkSimpleSlottingVar = atomically $ newTVar $ SimpleSlottingState $ unflattenSlotId 0
+mkSimpleSlottingStateVar :: (MonadIO m, HasConfiguration) => m SimpleSlottingStateVar
```

Comment:
Right I see, indeed the `HasConfiguration` is needed because `unflattenSlotId` uses the `k` parameter from `protocolConstants`. No action required, I'm just thinking aloud.